server/index.js

```diff
@@ -1,9 +1,27 @@
+// read env vars from local .env file
+const dotenv = require('dotenv');
+dotenv.config();
+
+// check for required env vars declared in app.json
+const fs = require('fs');
+const appJSON = fs.readFileSync('app.json', 'utf8');
+const appConf = JSON.parse(appJSON);
+
+Object.keys(appConf.env).forEach((key) => {
+  if (process.env[key] === undefined) {
+    console.error(key, 'is undefined!');
+    console.error('You should set all env vars!');
+    console.error('For details check app.json and README');
+    process.exit();
+  }
+});
```

Comment:
I think it would be better if it would list all the keys that are unset, instead of just the first one